src/Kunstmaan/NodeBundle/Resources/views/NodeAdmin/Modals/_pagetemplate.html.twig

```diff
@@ -0,0 +1,34 @@
+{% import _self as macros %}
+
+{% for row in pageTemplate.rows %}
+<div class="row">
```

Comment:
You should use [whitespace control modifier](http://vvv.tobiassjosten.net/symfony/trim-and-remove-twig-whitespace/).